schema_based_on_diagram.sql.sql

```diff
@@ -1 +1,46 @@
-/**All db schema*/
\ No newline at end of file
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(250) NOT NULL,
+    date_of_birth DATE
+);
+
+CREATE TABLE history_treatment_id (
+  id INT PRIMARY KEY
+);
```

Comment:
- [ ] Great job creating the "join table"! However, it should have two references, one for `medical_history`'s key and another one for `treatment`'s key in order to have a many-to-many relationship.

Reply:
okay noted